src/error.rs

```diff
@@ -102,7 +103,9 @@ impl<'tcx> Error for EvalError<'tcx> {
             EvalError::ModifiedConstantMemory =>
                 "tried to modify constant memory",
             EvalError::AssumptionNotHeld =>
-                "`assume` argument was false"
+                "`assume` argument was false",
+            EvalError::Assembler =>
+                "cannot evaluate assembler code",
```

Comment:
I decided to try out the "maintainers can push to PR branches" GitHub feature to fix this myself, since @oli-obk is probably asleep.